SAKPaaS/src/app/components/user-card/user-card.component.html

```diff
@@ -1,25 +1,31 @@
 <mat-card *ngIf="authService.getUser() | async as user; else login" id="user-card">
```

Comment:
Dadurch, dass du das `ngIf` ganz außen machst, hast du z.B. das Expand-Icon doppelt. Ich würde das `*ngIf` in `.content` runterschieben.